components/layout/blog-post/blog-post.hbs

```diff
@@ -0,0 +1,244 @@
+<div class="section--bg-prim-blue">
+
+    <div class="blog-post-hero section__inner">
+        <h1 class="blog-post-hero__title">If your team isn’t learning,
+            they’re falling behind. Fix it.</h1>
+        <div class="blog-post-hero__blog-author">
+            <div class="blog-post-hero__blog-author-img-container">
+                <img class="blog-post-hero__blog-author-img-container__author-image"
+                    src="http://placekitten.com/300/300" />
+            </div>
+            <div class="blog-post-hero__blog-author-info">
+                <span class="blog-post-hero__blog-author-info-name blog-author-name">By <a href="#">Meow
```

Comment:
Do we need to be declaring a new block called `blog-author-name` here? Can we limit it to just `blog-post-hero__author-name`?

Reply:
A mistake. Class name is removed.